operators/tensorboard-controller/tox.ini

```diff
@@ -19,6 +25,7 @@ commands =
 deps =
 	black==20.8b1
 	flake8
+	flake8-copyright<0.3
```

Comment:
Let's keep consistency, I've seen you've listed these deps in `test-req...txt` [here](https://github.com/canonical/kubeflow-volumes-operator/pull/8) and [here](https://github.com/canonical/admission-webhook-operator/pull/8).

Reply:
Yes you are right, fixed it.